sematic/ui/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "ui",
   "version": "0.2.0",
   "private": true,
+  "dependencies": {
```

Comment:
Maybe you wanted to install these in the `main` package, please double-check.

Also, you might want to install the corresponding @types packages in `devDependencies`.